tools/misc/rdiff-backup-wrap

```diff
@@ -35,14 +35,20 @@ case "$SOURCE" in
 esac
 
 # Pick the right version of rdiff-backup
-if [ $REMOTE_SOURCE -eq 0 ]; then
-    CMD="rdiff-backup2"
-else
+CMD=rdiff-backup	# default if nothing else is found
+
+for cmd in rdiff-backup-1 rdiff-backup2
+do
+	if command -v $cmd 2>&1 > /dev/null
+	then
+		CMD=$cmd
+	fi
+done
+
+if [ $REMOTE_SOURCE -eq 1 ]; then
     if rdiff-backup --test-server "$SOURCE" 2>&1 > /dev/null; then
         CMD="rdiff-backup"
```

Comment:
```suggestion
        CMD="rdiff-backup"
    else
        # because the version we found previously might also not work
        if ! $CMD --test-server "$SOURCE" 2>&1 > /dev/null; then
            echo "ERROR:   Couldn't find a working version of rdiff-backup, tried 'rdiff-backup' and '$CMD'" >&2
            exit 1
        fi
```
I didn't test it but it seems worth a try.

Reply:
Okay, I'll add that.  Previously such a test didn't exist, but this will make it more bulletproof.